src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,341 @@
+class Guides::Frontend::Internationalization < GuideAction
+  guide_route "/frontend/internationalization"
+
+  def self.title
+    "Internationalization"
+  end
+
+  def markdown : String
+    <<-MD
+    **Working with multiple languages**
+
+    If these steps are done in oder then Lucky should continue to compile (& be usable/testable) with each change.
+
+    ## Step 1 - Add i18n shard
+
+    ```
+    dependencies:
+      i18n:
+        github: vladfaust/i18n.cr
+        version: ~> 0.1.1
+    ```
+
+    add to the end of `shards.cr` file with the new requirements
+    ```
+    # shards.cr
+    # ...
+    require "i18n"
+    require "i18n/backends/yaml"
+    ```
+
+    and of course install the shard
+    ```
+    shards install
+    ```
+
+    ## Step 2 - Add localization ymls
+
+    first make a locales folder:
+    ```
+    mkdir config/locales
+    ```
+
+    then add at least one localization: i.e. English
+    ```
+    # config/locales/en.yml
+    en:
+      auth:
+        sign_in: "Sign-In"
+        sign_up: "Sign-up"
+        update_pwd: "Update Password"
+        reset_pwd_request: "Reset Password"
+      default:
+        page_name: Welcome
+      me:
+        email: E-Mail
+        profile: Profile
+        next: "Next, you may want to"
+        auth_guides: "Check out the Authentication Guides"
+        modify_page: "Modify this page"
+        after_signin: "Change where you go after sign in"
+    ```
+
+    and for example additional ones as needed: i.e. German
+    ```
+    # config/locales/de.yml
+    de:
+      auth:
+        sign_in: Anmeldung
+        sign_up: Anmelden
+        sign_out: Abmelden
+        update_pwd: "Kennwort aktualisieren"
+        reset_pwd_request: "Kennwort zurücksetzen"
+      default:
+        page_title: Wilkommen
+      me:
+        email: Mail
+        profile: Profil
+        next: "Zunächst bitte kontrolliere"
+        auth_guides: "Bitte kontrolliere die 'Authentication Guides'"
+        modify_page: "Um diese Seite zu Ändern"
+        after_signin: "Eine andere Seite nach anmelden"
+    ```
+
+    **NOTE:** All lang yml files need all the same keys defined. I18n shard is pretty finicky and its error messages aren't very helpful _(expect a little frustation getting the ymls correct and debugged)._
```

Comment:
```suggestion
   > Be sure that all lang yml files contain the same keys. If there's no translation for that key, just leave the value blank.
```

For these type of notes, we generally use the blockquotes. I'd also like to keep the connotation of the message on a more positive note. So I'm not completely sold on the suggested wording, but hopefully that gives you an idea of the direction. Thanks!

Reply:
Thanks for the suggestion. Updated